src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicRemoteExecutorTests.cs

```diff
@@ -22,7 +22,7 @@ public MsQuicRemoteExecutorTests()
         [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
         public void SslKeyLogFile_IsCreatedAndFilled()
         {
-            if (PlatformDetection.IsReleaseRuntime)
+            if (PlatformDetection.IsReleaseLibrary(typeof(QuicConnection).Assembly))
             {
                 throw new SkipTestException("Retrieving SSL secrets is not supported in Release mode.");
```

Comment:
This is nitpicking, but since there's also 'checked' build possibility, I feel it might be better to check `!IsDebugLibrary` instead of `IsReleaseLibrary` (I see that implementation-wise this is the same, but it would make a bit more sense to me)


```suggestion
            if (!PlatformDetection.IsDebugLibrary(typeof(QuicConnection).Assembly))
            {
                throw new SkipTestException("Retrieving SSL secrets is supported only in Debug mode.");
```

Reply:
Baah, I spent to much time looking at that 😂 you can disregard since this is merged already

Reply:
There's no "checked" version of library, that applies only to CLR.